frontend/Cargo.toml

```diff
@@ -1,18 +1,22 @@
 [package]
-name = "trunk-template"
+name = "flunk_website"
 version = "0.1.0"
 edition = "2021"
-description = "Template for starting a Yew project using Trunk"
+description = "Frontend for Flunk Website"
 readme = "README.md"
-repository = "https://github.com/yewstack/yew-trunk-minimal-template"
+repository = "https://github.com/LukeBrandon/fluke"
 license = "MIT OR Apache-2.0"
 keywords = ["yew", "trunk"]
 categories = ["gui", "wasm", "web-programming"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
+log = "0.4.6"
 yew = { version="0.20", features=["csr"] }
-yew-router = "0.15.0"
-yew_styles = { version="0.11", features=["layouts", "button"] }
-log = "0.4"
+yew-router = "0.17"
 wasm-logger = "0.2"
+wasm-bindgen = "0.2.84"
+web-sys = "0.3.61"
+wasm-bindgen-futures = "0.4"
+serde = "1.0.152"
+serde_json = "1.0.94"
```

Comment:
We already define these in common, I'm not sure if we need them here or if we just abolish the common folder entirely. I've seen a few repos use it